sd

```diff
@@ -115,8 +115,18 @@ sd() {
   point_path="$sdd/$requested_point"
 
   if [[ ! -L "$point_path" ]]; then
-    echo "Can't shift to point '$requested_point' because it doesn't exist."
+
+    type fzf >/dev/null 2>&1 || {
+    echo  "Can't shift to point '$requested_point' because it doesn't exist."
+    echo  "Also fzf issn't in path so we can't fuzzy find a match, sorry."
     return 1
+    }
+    point_path="$sdd/$(find "$sdd" -maxdepth 1 -type l -printf "%f\n"|fzf -q "$*")"
```

Comment:
The OSX version of `find` does not have the printf option. Can you do this another way?